vault/logical_system_paths.go

```diff
@@ -3658,7 +3658,7 @@ func (b *SystemBackend) policyPaths() []*framework.Path {
 
 			DisplayAttrs: &framework.DisplayAttributes{
 				OperationPrefix: "policies",
-				OperationVerb:   "list",
+				OperationSuffix: "acl-policy-list2", // this endpoint duplicates /sys/policies/acl
```

Comment:
I guess we could do that, but it's not clear to me why it would be considered better?

Personally, I like not setting OperationVerb to `list` on `ReadOperation`s (it feels like it adds confusion), and I like being able to express once, that the entire `sys/policy` endpoint duplicates `sys/policies/acl`, rather than needing to repeat it for each operation.

The Operation IDs:

| Current | With suggestion |
| --- | --- |
| policies-read-acl-policy-list2 | policies-list-acl-policies2 |
| policies-list-acl-policy-list2 | policies-list-acl-policies3 |

I wasn't trying hard to make the operation ID nice because it's going to be excluded anyway - I guess it comes down to whether you prefer it being more revealing which operation (read/list) each one belongs to, and that it's a separate endpoint, vs. making the operation ID match the ID of the other operation that is being duplicated.

Should I consider the latter, i.e. making the operation ID match the ID of the other operation that is being duplicated, to be the project's preferred style, in this situation?

Reply:
It's just a convention that I've been sticking with when naming things. When there are two or more operations that do exactly the same thing (despite one being ReadOperation and one being ListOperation in this case), they would be mapped to the same name with a numerical suffix. I still believe we should have a more declarative way of excluding them from the API libraries. Will probably put up a draft PR sometime this week :)